region/lowernorfair/east/Pillar Room.json

```diff
@@ -608,17 +695,131 @@
       ]
     },
     {
-      "id": 17,
       "link": [2, 1],
-      "name": "Shinespark",
+      "name": "Come In With Spark",
+      "entranceCondition": {
+        "comeInWithSpark": {
+          "position": "top"
+        }
+      },
+      "requires": [
+        {"heatFrames": 79},
+        {"or": [
+          {"and": [
+            {"shinespark": {"frames": 79, "excessFrames": 5}},
+            {"heatFrames": 105}
+          ]},
+          {"and": [
+            {"shinespark": {"frames": 79, "excessFrames": 16}},
+            "canCarefulJump",
+            {"or": [
+              "canTrickyJump",
```

Comment:
I can't tell if  this frame count lands on the pillar or ends slightly after it.  Landing on the pillar and avoiding any acid damage is an insane jump or trickyJump+walljump.

Reply:
It can cover both cases. If your spark ends close enough to the pillar you can land on it. Otherwise you can land on either the high part of the floor or on the door platform. Avoiding all acid damage seems possible in all cases. 

Adjusted the requirements for the jump from the pillar as you suggested. Also added a HiJump alternative to walljump for escaping the acid in the comeInShinecharged/comeInShinecharging cases, where the acid will be higher; in the  comeInWithSpark case, the acid is low enough that it doesn't affect your jump, so getting out is free.